internal/vargs/vargs.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
```

Comment:
The name vargs seems odd. It's usually either "vars" or "args". If it's a portmanteau of "var" and "arg" then "vararg" actually means variable number of arguments.

Pick one: vars, args, or params. And the flag names should probably be prefixed by the package name.